From: Hannes Reinecke <hare@suse.de>
Date: Fri, 5 Jun 2015 09:48:52 +0200
Subject: firmware: iBFT firmware is also supported on EFI machines with ACPI
References: bsc#932047
Patch-Mainline: n/a

The iBFT firmware tables can also be specified via ACPI tables
when using EFI firmware. The 'iscsi_ibft_find' module is only
for legacy X86 BIOS, so it needs to be skipped for all other
architectures.

Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 drivers/firmware/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/firmware/Kconfig b/drivers/firmware/Kconfig
index 25e88a5..3d934c7 100644
--- a/drivers/firmware/Kconfig
+++ b/drivers/firmware/Kconfig
@@ -139,7 +139,7 @@ config ISCSI_IBFT_FIND
 config ISCSI_IBFT
 	tristate "iSCSI Boot Firmware Table Attributes module"
 	select ISCSI_BOOT_SYSFS
-	depends on ISCSI_IBFT_FIND && SCSI && SCSI_LOWLEVEL
+	depends on (ISCSI_IBFT_FIND || ACPI) && SCSI && SCSI_LOWLEVEL
 	default	n
 	help
 	  This option enables support for detection and exposing of iSCSI
